### ISSUE TITLE
MD037 does not detect spaces in `***` inlines when used to apply emphasis+strong together

### ISSUE THREAD

**Issue by TheJaredWilcurt** [DavidAnson/markdownlint#218]
**MD037** only handles strong, not em or both. This is inconsistent with how it is documented.

* https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md037---spaces-inside-emphasis-markers

~~❌ This does not throw an error~~  
✔️ This throws error:

```md
Text * text *.
```

✔️ This throws error:

```md
Text ** text **.
```

❌ This does not throw an error

```md
Text *** text ***.
```
